docs/fsharp/language-reference/computation-expressions.md

```diff
@@ -231,24 +231,26 @@ The following table describes methods that can be used in a workflow builder cla
 |**Method**|**Typical signature(s)**|**Description**|
 |----|----|----|
 |`Bind`|`M<'T> * ('T -> M<'U>) -> M<'U>`|Called for `let!` and `do!` in computation expressions.|
```

Comment:
Just to mention that BindReturn should be added to this list, https://github.com/dsyme/fsharp-presentations/blob/master/design-notes/rethinking-applicatives.md

Reply:
Yeah, I never got around to that one: https://github.com/dotnet/docs/issues/21272

Note that it's not just about updating the table, but also about explaining what it does and having a meaningful example.